internal/pkg/template/env.go

```diff
@@ -103,10 +104,17 @@ type EnvOpts struct {
 	AllowVPCIngress          bool
 	Telemetry                *Telemetry
 
-	LatestVersion      string
+	CDNConfig CDNConfig
```

Comment:
```suggestion
	CDNConfig *CDNConfig
```
Then maybe we don't need `EnableCDN bool` at all?